util/publish-to-confluence.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const { prompt } = require('inquirer');
+const path = require('path');
+const fs = require('fs');
+const fetch = require('node-fetch');
+const showdown = require('showdown');
+const error = require('../lib/error');
+
+const converter = new showdown.Converter(
+  {
+    parseImgDimensions: true,
+    simplifiedAutoLink: true,
+    tables: true
+  }
+);
+
+const CONFLUENCE_PAGES = './confluence-pages.json';
+
+const CONFLUENCE_DOMAIN = process.env.CONFLUENCE_DOMAIN;
+const CONFLUENCE_SPACE = process.env.CONFLUENCE_SPACE;
+const CONFLUENCE_USER = process.env.CONFLUENCE_USER;
+const CONFLUENCE_PASS = process.env.CONFLUENCE_PASS;
+
+async function gatherCreds () {
+  const answer = await prompt([
```

Comment:
I think it would be nice to detect the presence of the `process.env` values and if not present, ask for this info.

Reply:
i actually like the prompts to serve as a way to overwrite env values